src/uproot/extras.py

```diff
@@ -23,21 +22,21 @@ def awkward():
     """
     try:
         import awkward
-    except ImportError:
-        raise ImportError(
+    except ModuleNotFoundError:
+        raise ModuleNotFoundError(
             """install the 'awkward' package with:
 
     pip install awkward
 
 Alternatively, you can use ``library="np"`` or globally set ``uproot.default_library``
 to output as NumPy arrays, rather than Awkward arrays.
 """
-        )
+        ) from None
```

Comment:
This is still an open question, actually, didn't realize I put one in. Should we hide the original exception (like Python 2), or should it be chained on? I'm thinking hiding it _might_ be okay for these? It does reduce the burden on the user, but it can hide, for example, if a failure is due to some problem other than a simple installation. But in that rare case, then a user could just try importing it.

Reply:
I haven't been hiding any exceptions. The chained exception output is verbose, but the only important bit for a user (non-debugger) is at the end. I'd think Python users are used to only looking at the end of an exception stack trace.

Reply:
Let's keep chaining exceptions. Changing that (if we do it at all) doesn't have to be done at the same time as this.

Reply:
Python 2 does not chain exceptions. For Python 2, you would never see the exception that caused the one you throw. Python 3 does, and does it by default. So the question would be would this be cleaner restoring the Python 2 output, or should we embrace the Python 3 output? :)

Yes, I can remove it, didn't realize I changed one. I started to change it, then decided to ask, and seems like I left this one in.

```suggestion
        )
```

Reply:
I know: only Python 3 shows the chained stack trace, so there had been a difference between how it looked in Python 2 and 3. As a user of Python 3, I preferred the chains, and I'm glad they're the default.

By "not changing this," I mean keep the chained exceptions: i.e. not change the Python 3 behavior.